hack/verify-gofmt.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-go_files=$( find . -name '*.go' -print )
+go_files=$( find . -name '*.go' -not -path '*/vendor/*' -print )
```

Comment:
I suppose the whole operator is not needed to be verified for the go format as it has its own Makefile targets for this.
```suggestion
go_files=$( find . -name '*.go' -not -path '*/operator/*' -print )
```